scripts/recan_bundle_tdr.py

```diff
@@ -263,20 +268,26 @@ def dump_tables(bundle: TDRBundle) -> MutableJSON:
     }
 
 
-def add_supp_files(bundle: TDRBundle, *, num_files: int) -> None:
+def add_supp_files(bundle: TDRBundle,
+                   *,
+                   num_files: int,
+                   version: str
+                   ) -> None:
     links_json = bundle.metadata_files['links.json']['links']
     links_manifest = one(e for e in bundle.manifest if e['name'] == 'links.json')
     project_id = bundle.metadata_files['project_0.json']['provenance']['document_id']
 
-    metadata_ids = sorted(random_uuid() for _ in range(num_files))
+    # Entities must be sorted by UUID for the incrementing document names to
+    # match the plugin's output.
+    metadata_ids = sorted(deterministic_uuid(bundle.uuid, str(i) + 'supp file')
+                          for i in range(num_files))
 
     for i, metadata_id in enumerate(metadata_ids):
```

Comment:
This loop would then read 

```suggestion
    for document_name, metadata_id in enumerate(metadata_ids):
```